components/sms/README.md

```diff
@@ -1,17 +1,18 @@
 # Overview
-Use Pipedream's SMS app to send a message from a workflow to the [verified phone number for your account](https://pipedream.com/settings) and incorporate SMS into whatever you're building with any of the 900+ apps that are available on Pipedream. Use the pre-built action to send an SMS without writing any code, or for more control pass a string to `$.send.sms()` in a Node.js code step. 
+
+Use Pipedream's SMS app to send a message from a workflow to the [verified phone number for your account](https://pipedream.com/settings) and incorporate SMS into whatever you're building with any of the 1000+ apps that are available on Pipedream. Use the pre-built action to send an SMS without writing any code, or for more control pass a string to `$.send.sms()` in a Node.js code step.
 
 **Important:** Pipedream SMS is currently in alpha. You may send up to 10 messages per day from workflows in a personal account (organization accounts are not currently supported).
 
 # Getting Started
 
 ## Adding an SMS number
 
-To get started, go to https://pipedream.com/alpha and enable the SMS feature.
+To get started, go to <https://pipedream.com/alpha> and enable the SMS feature.
 
 Next, go to your [account settings](https://pipedream.com/settings) and enter and save an **SMS Number**:
 
-![Enter an SMS number](https://res.cloudinary.com/pipedreamin/image/upload/v1663724994/marketplace/apps/sms/sms-account-settings_uhsvqa.png)
+![Enter an SMS number (only US numbers are supported)](https://res.cloudinary.com/pipedreamin/image/upload/v1663724994/marketplace/apps/sms/sms-account-settings_uhsvqa.png)
```

Comment:
Is "(only US numbers are supported)" intended to be in the image's alternate text only?